augraphy/augmentations/watermark.py

```diff
@@ -194,7 +194,9 @@ def apply_watermark(self, watermark_foreground, image):
             # set removed contours to white
             if len(new_watermark_foreground.shape) > 2:
                 image_eroded = cv2.cvtColor(image_eroded, cv2.COLOR_GRAY2BGR)
+
             new_watermark_foreground[image_eroded == 0] = 255
+            image[new_watermark_foreground < 255] = 255
```

Comment:
With this update, watermark and the text in the image will be obfuscating each other, for example:

![image](https://user-images.githubusercontent.com/52521979/165297073-84fccded-4e10-41a5-86de-c8af7677eb2d.png)
![image](https://user-images.githubusercontent.com/52521979/165297141-f9f8c468-dc9d-4547-b98e-4a8a0e077e63.png)

![image](https://user-images.githubusercontent.com/52521979/165297266-18ddc1a8-b382-44a6-9f6c-bb8d819f0135.png)
![image](https://user-images.githubusercontent.com/52521979/165297304-856ed478-640a-44e7-9ba8-cc4f6932b22c.png)